lib/elastic_whenever/task/rule.rb

```diff
@@ -21,20 +23,23 @@ def self.convert(option, task)
         self.new(
           option,
           name: rule_name(option.identifier, task.expression, task.commands),
-          expression: task.expression
+          expression: task.expression,
+          description: rule_description(option.identifier, task.expression, task.commands)
         )
       end
 
-      def initialize(option, name:, expression:)
+      def initialize(option, name:, expression:, description: "")
```

Comment:
Is this default value required?

Reply:
Not really. I was just being cautious in case this is used elsewhere. Happy to remove.